doc/nrf/config_and_build/bootloaders/bootloader_signature_keys.rst

```diff
@@ -148,11 +137,7 @@ See :kconfig:option:`CONFIG_SB_PUBLIC_KEY_FILES` for details on how this mechani
 
 You can add this feature to your own project and check its functionality as follows:
 
-1. Use the :ref:`bootloader Python script <ug_fw_update_keys_python>` to generate two or more private keys for the application and extract a public key for each one:
-
-   .. code-block:: console
-
-      python3 nrf/scripts/bootloader/keygen.py --public -i priv.pem -o pub.pem
+1. Generate two or more private keys for the application and extract a public key for each one.
```

Comment:
If this just says "Generate two private keys and extract a public key for each" without referring to how, linking to how or explaining how, some readers will be confused on how to do it. 
I think it would be sufficient to add something like 
```suggestion
1. Generate two or more private keys for the application and extract a public key for each one. For example using openssl.
```
(Or whatever other tool we want to recommend)